### PR TITLE
docs: add examples

### PR DESCRIPTION
**Pull request by YanniKouloumbis**
- Add an `Examples` section to the `Docs` section for template code examples

Resolves WIN-60

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,6 +144,10 @@ This **does not guarantee that the length of the return result will equal `numOu
 The `onStreamResult` handler is similar. You should rely on the promise resolution and only use this
 handler to improve UX, since not all models and config options support it.
 
+### Examples
+
+- [Next.js Window AI](https://github.com/YanniKouloumbis/next-js-window-ai) - A Next.js app that demonstrates how to use Window AI in a chat application. ([Live Demo](https://next-js-window-ai.vercel.app/))
```
Comment (alexanderatallah) [inline README.md:149]: can we add another example linking to https://robot-companion.vercel.app/ ?

**Review by alexanderatallah — COMMENTED** [review]
commented

**Review by alexanderatallah — APPROVED** [review]
preemptive approval

**Comment by YanniKouloumbis** [comment]
Addressed & merging!

**Pull request merged**